i18n/sample/package.i18n.json

```diff
@@ -71,7 +71,7 @@
     "rename": "Rename",
     "copy": "Copy",
     "paste": "Paste",
-    "hMigrateDataSet": "Migrate Data Set",
+    "migrateDataSet": "Migrate Data Set",
```

Comment:
Was there a rationale regarding the changing of field names?. This was representing the (Hierarchical) Migrate DFSMShsm command and although I prefer the abridged version we run the risk of reducing the meaning of the chosen names particulalrly to z/OS developers looking at the implementation.

Reply:
Okay, I wasn't sure why it was there so I removed it. We can put it back.

Reply:
No problem. As I say I prefer the 'h' less version but thought I worth calling out that sometimes the fields/variables have been so named to impart additional meaning. So this relates to HMigrate and another new function HRecall will be added soon